### DISCUSSION TITLE
Reboot with fancybox open

### DISCUSSION THREAD

**Discussion by MyNameTony1000** [fancyapps/ui#414] [Q&A] [answered]
Hello, my js code is split into 2 files (libraries and my code). If you reload the page with the "fancybox" open, it will load and the "fancybox" will be opened with the default settings, not mine. I'd like to avoid this behavior (similar queries #159 and #6 ).
Is it possible to somehow influence this behavior? 
For example, to make an initialization on my attribute or to make a check on the opening of the window and if it is open - just close it? Is there a smarter way to get around this problem?
Wouldn't like to change the location of my code for the "fancybox" setup, but obviously the problem is that my code runs later than the default init code.
I would be very grateful for help in this matter)))

**Comment by fancyapps** [comment, marked as answer]
Hi,

This is one of the reasons why Fancybox5 does not auto-bind, see https://fancyapps.com/resources/upgrading/#fancybox
For Fancybox4, you could, for example, put `<script>Fancybox.destroy();</script>` right after including JS file.

**Reply by MyNameTony1000** [reply]
it works for me, thank you very much))
